### DISCUSSION TITLE
Share Your Goodwill | /en/2023/04/share-your-goodwill/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1489] [Comments]
Recently I often hear my son sing a song at home that sounds touching to me. The incomplete lyrics from his mouth included sentences like &ldquo;remember there are people not as fortunate as &hellip;

https://yihui.org/en/2023/04/share-your-goodwill/

**Comment by liechi** [comment]
这首歌让我想起 Phil Collins 的 Another Day In Paradise，那似乎是 goodwill 的成人版。小时候觉得对、理所当然的事情，长大后做起来往往却不那么简单。我不算非常铁石心肠的人，但每次听《天堂里的另一天》时，都会记起曾经漠然的几次，感到羞愧。

**Reply by yihui** [reply]
我现在还记得大约 2010 或 11 年的时候，有次我在超市买完东西后推车走向收银台时，用余光看到一位老太太从我的右边走过来，如果按我正常的步速，她大概会和我同时走到收银台。当时我脑子下意识的反应是，有人要走到我前面结账了，所以我要快走几步，于是我就加快步伐冲在了她前面结账了。

我觉得人性里总是有一些时刻会浮现自私自利和漠然的。我们没必要有道德洁癖。

**Comment by JinjiPang** [comment]
新的网页设计的很好看，我刚看到，然后一打眼就看到了这篇英文Blog. 我感觉有的时候我也是控制不住的自私，总想着节省自己的时间，尽快的做完一些事情。周末开车去奥马哈的时候我还很感慨，因为有很多车进村，所以右侧车道都是车在排队，但是同侧的左车道却很空旷，但是也没有人从左车道开过去再并入右车道。所以我有时候还挺佩服美国人的，他们确实不那么“着急”。

**Reply by yihui** [reply]
这些年来我的网页设计风格正在无限逼近白纸模式……这个新设计的主题其实有两个特色我都没用在自己的网站上。一个是[显示日志摘要](https://paged.yihui.org/blog/)，另一个是[文档模式中显示目录](https://paged.yihui.org/docs/)。我自己的日志量太大了，要是用这些模式展示的话，日志列表页面会变得无比之长。

在中国开车确实是大家都太抢了，不过确实也是人口众多没办法。美国的乡下人口稀少，大家都习惯了慢节奏，没啥需要争抢的，加上西方文化里本来也是（绅士）风度很重要，所以就规规矩矩排队等候了。
